Tools/ExceptionFinderTool/Program.cs

```diff
@@ -17,7 +17,9 @@ public static void Main()
         {
             var stringBuilder = new StringBuilder();
 
-            foreach (var exceptionType in GetAllAssemblies()
+            foreach (var exceptionType in AppDomain
+                .CurrentDomain
+                .GetAssemblies()
```

Comment:
Is this change relevant to non-nullability?

Reply:
Could have done this in another PR. Removed some unused code only relevant to `NETCOREAPP1_1`.